test/e2e/multi-zones/monorepo/apps/main/pages/index.js

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link'
+import CustomLink from '@acme/ui/link'
```

Comment:
Shouldn't this package be configued under `transpilePackages` so that webpack can inline `proceess.env.__NEXT` values correctly?